packages/core/src/renderer/components/+cluster/cluster-overview.tsx

```diff
@@ -121,6 +122,6 @@ export const ClusterOverview = withInjectables<Dependencies>(NonInjectedClusterO
     podStore: di.inject(podStoreInjectable),
     eventStore: di.inject(eventStoreInjectable),
     nodeStore: di.inject(nodeStoreInjectable),
-    uiBlocks: di.injectMany(clusterOverviewUIBlockInjectionToken),
+    uiBlocks: di.inject(computedInjectManyInjectable)(clusterOverviewUIBlockInjectionToken),
```

Comment:
Shout if you want `di.computedInjectMany(...)`...

Reply:
That'd be useful!